cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -50,7 +50,7 @@ foreach(x ${boost_libs})
 endforeach()
 
 if("@MSVC@")
-  set(env_cmd "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@")
+  set(env_cmd "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@" && cd "@HUNTER_PACKAGE_SOURCE_DIR@" &&)
```

Comment:
`&&` should not be used, see https://github.com/ruslo/hunter/issues/246

Reply:
Would it be ok to generate a .bat or .sh file run during COMMAND step ?

Reply:
> Would it be ok to generate a .bat or .sh file

I don't see the point of using platform specific scripts when we can handle it with cross-platform CMake scripts.

Reply:
I don't see how to handle it with CMake scripts since COMMAND step nullify the environment and && is forbidden

Reply:
You can generate them before ExternalProject and use copy on each stage.

This is how it's implemented in Qt:
* https://github.com/ruslo/hunter/blob/0e0e1f87c7b8ef7ace8673cb9380590ce5ee5f5c/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L386-L390
* https://github.com/ruslo/hunter/blob/0e0e1f87c7b8ef7ace8673cb9380590ce5ee5f5c/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L402-L406
* https://github.com/ruslo/hunter/blob/0e0e1f87c7b8ef7ace8673cb9380590ce5ee5f5c/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L442-L470

It would be nice to generalize this approach, I'm opened for suggestions here.